src/encoding.h

```diff
@@ -0,0 +1,4467 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+/* Copyright (c) 2023 RISC-V International */
+
+/*
+ * This file is auto-generated by running 'make' in
+ * https://github.com/riscv/riscv-opcodes (9226b0d)
+ * and it is temporarily used by RVV only
+ */
```

Comment:
Related: https://github.com/sysprog21/rv32emu/issues/103

Do you think it makes sense to replace the existing hand-written RISC-V instruction decoder with the generated one?

Reply:
I think the current one is good for education, especially the comments are valuable.
However I think having a table to clearly store the instructions in human-readable form is good for the future development.

Reply:
But this encoding.h is only a temporary header file for me to decode rvv instructions.